tests/e2e/pageobjects/login/RegularUserOcpCheLoginPage.ts

```diff
@@ -20,34 +21,50 @@ import { DriverHelper } from '../../utils/DriverHelper';
 
 @injectable()
 export class RegularUserOcpCheLoginPage implements ICheLoginPage {
+  private readonly OPEN_SHIFT_LOGIN_LANDING_PAGE_LOCATOR: string = `//div[@class='panel-login']`;
+  private readonly OPEN_SHIFT_LOGIN_LANDING_PAGE_BUTTON_LOCATOR: string = `${this.OPEN_SHIFT_LOGIN_LANDING_PAGE_LOCATOR}/div[contains(@class, 'panel-content')]/form/button`;
 
-    private readonly OPEN_SHIFT_LOGIN_LANDING_PAGE_LOCATOR: string = `//div[@class='panel-login']`;
-    private readonly OPEN_SHIFT_LOGIN_LANDING_PAGE_BUTTON_LOCATOR: string = `${this.OPEN_SHIFT_LOGIN_LANDING_PAGE_LOCATOR}/div[contains(@class, 'panel-content')]/form/button`;
+  constructor(
+    @inject(CLASSES.OcpLoginPage) private readonly ocpLogin: OcpLoginPage,
+    @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper,
+    @inject(CLASSES.OauthPage) private readonly oauthPage: OauthPage
+  ) {}
 
-    constructor(
-        @inject(CLASSES.OcpLoginPage) private readonly ocpLogin: OcpLoginPage,
-        @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper) { }
+  async login(): Promise<void> {
+    Logger.debug('RegularUserOcpCheLoginPage.login');
 
-    async login(): Promise<void> {
-        Logger.debug('RegularUserOcpCheLoginPage.login');
+    Logger.debug(
+      'RegularUserOcpCheLoginPage.login wait for LogInWithOpenShift page and click button'
+    );
+    await this.driverHelper.waitPresence(
+      By.xpath(this.OPEN_SHIFT_LOGIN_LANDING_PAGE_LOCATOR),
+      TimeoutConstants.TS_SELENIUM_LOAD_PAGE_TIMEOUT
+    );
```

Comment:
![image](https://github.com/eclipse/che/assets/72726250/9e867c4d-94e6-4e78-a6b7-29a3824fdc4a)

I am not sure all this new lines necessary. Can you switch them back?